src/data/set/lattice.lean

```diff
@@ -1050,7 +1050,9 @@ ht.sup_right hu
 theorem disjoint_diff {a b : set α} : disjoint a (b \ a) :=
 disjoint_iff.2 (inter_diff_self _ _)
 
-theorem disjoint_compl (s : set α) : disjoint s sᶜ := assume a ⟨h₁, h₂⟩, h₂ h₁
```

Comment:
Could you edit the first post of the PR to mention this renaming, so that it shows up in search/the commit message?